controller/status.go

```diff
@@ -213,6 +213,13 @@ var (
 	StatusIndex = []StatusContext{
 		{
 			FleetCurrent:  "inactive",
+			FleetDesired:  "launched",
+			SystemdActive: "active",
+			SystemdSub:    "running",
+			Aggregated:    StatusRunning,
+		},
```

Comment:
Shouldn't we rather change this to `*`? https://github.com/giantswarm/inago/blob/fix-global-start-hang/controller/status.go#L271